src/ops/acme.rs

```diff
@@ -0,0 +1,49 @@
+use crate::msg;
+use crate::runtime::{JsRuntime, Op};
+use crate::utils::*;
+use flatbuffers::FlatBufferBuilder;
+use futures::Future;
+use libfly::*;
+
+pub fn op_validate_challenge(ptr: JsRuntime, base: &msg::Base, _raw: fly_buf) -> Box<Op> {
+  let cmd_id = base.cmd_id();
+  let msg = base.msg_as_acme_validate_challenge().unwrap();
+  let hostname = msg.hostname().unwrap().to_string();
+  let token = msg.token().unwrap().to_string();
+
+  let rt = ptr.to_runtime();
+
+  if rt.acme_store.is_none() {
+    return Box::new(odd_future("No acme store configured".to_string().into()));
+  }
+
+  let acme_store = rt.acme_store.as_ref().unwrap();
```

Comment:
I mention this in another comment, but I think this could be a global thing. Not sure that it matters if it's specific to an app or not.